src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedVideoConnection.java

```diff
@@ -102,11 +123,26 @@ public InitializeUploadResponse initializeUpload(
         Parameter.with(ACTION_KEY, INITIALIZE_UPLOAD));
   }
   
-  public List<String> uploadVideoFile(String filePath,
+  public List<String> uploadVideoFileFromURL(String videoFileURL,
```

Comment:
Likewise, this is very similar to the `uploadVideoFileFromDirectory` method. The only difference is converting a URL or file to a byte array. It would be cleaner to pass a byte array into one method that does the uploading of the video chunks